ui/src/Tags/GroupedTagFilterOptions.ts

```diff
@@ -0,0 +1,27 @@
+import {TagInterface} from './Tag.interface';
+import {FilterOption} from '../CommonTypes/Option';
+import {AllGroupedTagFilterOptions} from '../SortingAndFiltering/FilterConstants';
+
+export function addGroupedTagFilterOptions(
```

Comment:
Should this be in the FilterConstants.tsx file ? and maybe the file could get renamed. Because this is another function related to this allGroupedTagFilterOption that one day we will refactor...

Reply:
Or vice-versa. Maybe all the filter constants should go in here.